# SendGoal.srv
string target_name
float64 target_x
float64 target_y
float64 target_orientation_z
float64 target_orientation_w
---
bool success
